### PR TITLE
[T] New Icons of Design System

### PR DESCRIPTION
**Pull request by volcanioo**
Hey fellas, 
I updated the whole DS icons via `asset-import`. The script added new icons and updated some of the icons that need a typo update. 

### New Icons
| Small  | Big | 
| ------------- | ------------- |
| `contractIcon`, `validationIcon`, `thumbUpIcon` | `footballCelloIcon`, `videoFilmCameraIcon`, `euroChangeIcon`  | 

### Updated Icons
| Old Version  | New Version | Affected Products | 
| ------------- | ------------- | ------------- |
| `whatappIcon` | `whatsappIcon`  | @homeday-de/customer-experience-squad-frontend   | 

**my dear friends, please update the icons on your product (if it's in the updated icons section) when I deploy these changes**

We don't have a ticket for this.

### CODE REVIEW

File: src/assets/small-icons/index.js [hunk 1]
```diff
@@ -105,11 +107,12 @@ export { default as twitterIcon } from './ic_twitter.svg';
...
 export { default as userCircleIcon } from './ic_user-circle.svg';
 export { default as userIcon } from './ic_user.svg';
+export { default as validationIcon } from './ic_validation.svg';
 export { default as verifiedIcon } from './ic_verified.svg';
 export { default as visibilityOffIcon } from './ic_visibility-off.svg';
 export { default as visibilityOnIcon } from './ic_visibility-on.svg';
 export { default as voiceIcon } from './ic_voice.svg';
 export { default as warningIcon } from './ic_warning.svg';
-export { default as whatappIcon } from './ic_whatapp.svg';
+export { default as whatsappIcon } from './ic_whatsapp.svg';
```
Comment (viniciuskneves) [inline src/assets/small-icons/index.js:116]: Hey, I see this as a potential pitfall! We will be renaming the WhatsApp icon and we should release a major version afterwards!

`customer-app` uses it: https://github.com/homeday-de/customer-app/blob/7ebadee8334b054f4c0399f6b59b6294fc546ac6/src/components/evaluation-report/EvaluationReportSharing.vue#L110

@homeday-de/customer-experience-squad-frontend please be aware of that once updating the library.

File: src/assets/big-icons/index.js [hunk 2]
```diff
@@ -25,15 +25,16 @@ export { default as documentsLockIcon } from './ic_documents-lock.svg';
 export { default as documentsStatisticIcon } from './ic_documents-statistic.svg';
 export { default as elevatorIcon } from './ic_elevator.svg';
 export { default as envelopeMailLetterIcon } from './ic_envelope-mail-letter.svg';
+export { default as euroChangeIcon } from './ic_euro-change.svg';
```
Comment (viniciuskneves) [inline src/assets/big-icons/index.js:28]: You didn't mention this one in the description but it has also been added, I think @homeday-de/realtor-performance-squad-frontend uses it and can replace by blocks afterwards.

File: src/assets/small-icons/index.js [hunk 3]
```diff
@@ -97,6 +98,7 @@ export { default as starBorderIcon } from './ic_star-border.svg';
 export { default as starFilledIcon } from './ic_star-filled.svg';
 export { default as starHalfFilledIcon } from './ic_star-half-filled.svg';
 export { default as tasksIcon } from './ic_tasks.svg';
+export { default as thumbUpIcon } from './ic_thumb-up.svg';
```
Comment (viniciuskneves) [inline src/assets/small-icons/index.js:101]: This one has also been added but not listed in the PR description. I don't know where it is used but can definitely be replaced afterwards.

**Review by viniciuskneves — APPROVED** [review]
Nice one!

I just dropped few comments that we have to keep in mind but it looks good to go!

File: src/assets/big-icons/index.js (hunk 2 above)
Comment (volcanioo) [reply src/assets/big-icons/index.js:28]: I'll check the whole changes and write the description again, I'll be pinging the teams to update their stack after this merge.

File: src/assets/small-icons/index.js (hunk 3 above)
Comment (volcanioo) [reply src/assets/small-icons/index.js:101]: I updated the description.

File: src/assets/big-icons/index.js (hunk 2 above)
Comment (volcanioo) [reply src/assets/big-icons/index.js:28]: I updated the description.

File: src/assets/small-icons/index.js (hunk 1 above)
Comment (volcanioo) [reply src/assets/small-icons/index.js:116]: Hey man! First, thanks for your attention. I'll be sharing this with the team when I merge my changes with deploy. Second, how did you find the customer app's code line? Where did you search the `whatappIcon`? VSCode, github search or something else?

Comment (viniciuskneves) [reply src/assets/small-icons/index.js:116]: GitHub: https://github.com/homeday-de/customer-app/search?q=whatappIcon

**Review by metal-gogo — APPROVED** [review]

**Pull request merged**